app/assets/stylesheets/spam.css

```diff
@@ -0,0 +1,235 @@
+#wrapper {
+  padding-left: 10px;
+  transition: all .4s ease 0s;
+  height: 100%
+}
+#sidebar-wrapper {
+  margin-left: -15%;
+  transition: all .4s ease 0s;
+  position: fixed;
+  top: 50px;
+  right: 0;
+  left: 0;
+  width: 15%;
+  height: 100vh;
+  z-index: 4;
+}
+
+#page-content-wrapper {
+  width: 100%;
+  height: auto;
+}
+#wrapper.active {
+  padding-left: 100px;
+}
+#wrapper.active #sidebar-wrapper {
+  left: 15%;
+}
+
+.nav_top  {
+    position: fixed ;
+    top: 60px;
+    right: 0;
+    left: 0;
+    padding-left: 0;
+    height: 3.625rem;
+    z-index: 5;
+    width:100%;
+    font-size: 0.9rem;
+    box-shadow: 0 0.15rem 1.75rem 0 rgba(31, 45, 65, 0.15) !important;
+    background-color:white;
+    overflow: hidden;
+  }
+  .navbar_title{
+    width: 16vw;
+    padding-left: 1rem;
+    margin: 0;
+    font-size: 1.1rem;
+    font-weight: bold;
+  }
+
+  .sidenav {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    flex-wrap: nowrap;
+    font-size: 0.9rem;
+    padding-top: 3.625rem;
+    background-color: #fff;
+    color: #1f2d41;
+    box-shadow:  5px 0 5px -5px rgba(31, 45, 65, 0.15) !important;
+    
+  }
+  .sidenav-menu{
+      overflow-y: auto;
+      flex-grow: 1;
+  
+  }
+  .sidenav-menu .nav-link .nav-link-icon {
+    margin-right: 0.5rem;
+  }
+  .sidenav .sidenav-menu .nav {
+    flex-direction: column;
+    flex-wrap: nowrap;
+  }
+  .sidenav .sidenav-menu .nav .sidenav-menu-heading {
+    padding: 1.20rem 1.5rem 0.75rem;
+    font-size: 0.7rem;
+    font-weight: bold;
+    text-transform: uppercase;
+    letter-spacing: 0.05em;
+    color: #a2acba;
+  }
+  .sidenav .sidenav-menu .nav .nav-link {
+    display: flex;
+    align-items: center;
+    line-height: normal;
+    padding-top: 0.70rem;
+    padding-bottom: 0.70rem;
+    position: relative;
+    color: #1f2d41;
+  }
+  .sidenav .sidenav-menu .nav .nav-link .nav-link-icon {
+    font-size: 0.9rem;
+    padding-right: 0.5rem;
+    display: inline-flex;
+    color: #a2acba;
+  }
+  .sidenav .sidenav-menu .nav .nav-link.active {
+    font-weight: 600;
+    color: #0061f2;
+  }
+  .sidenav .sidenav-menu .nav .nav-link.active .nav-icon {
+    color: #303438;
+  }
+  #layoutSidenav_content {
+    margin-left:-6vw;
+    width:95vw;
+    top: 3.625rem;
+  }
+  #wrapper.active #layoutSidenav_content{
+    margin-left:0px;
+    width:83vw;
+  }
+  .page-header{
+  background-color: #0061f2;
+  background-image: linear-gradient(135deg, #0061f2 0%, rgba(105, 0, 199, 0.8) 100%);
```

Comment:
Hi, @keshavsethi - i think maybe we should just start with minimal, default styles that we already use on the site for now, as I'm not sure we yet want to introduce a lot of new visual styles without thinking through it carefully. You can see the style guide we use here, actually! https://publiclab.org/notes/warren/05-07-2019/introducing-a-draft-style-guide-for-public-lab

Thank you!! I think we're just thinking of colors, fonts, etc here. It's fine for this to be somewhat different from the rest of the site since it's an admin-only dashboard, but perhaps we should aim to develop only a minimum of new visual styles, what do you think?

Reply:
Sure, I was also thinking of changing the page header and make it as per the style guide. I will surely work on that. 

Thanks !!